espnet2/fst/lm_rescore.py

```diff
@@ -0,0 +1,187 @@
+from typing import List
```

Comment:
ditto.
please increase the test coverage.

Reply:
fixed. Now it's covered by ci/test_integration_espnet2.sh.